app/services/base_workflow_service.rb

```diff
@@ -1,35 +1,12 @@
 class BaseWorkflowService
+  delegate :applicant, :capital_summary, to: :assessment
+  delegate :liquid_capital_items, :main_home, :additional_properties, :vehicles, to: :capital_summary
+
+  attr_reader :assessment
```

Comment:
All the attributes assigned in `initialize` should have a getter set, so the getters can be used in child objects rather than the instance variable. So:
```ruby
attr_reader :assessment, :submission_date, :calculation_period
```
The great advantage of this is that it makes it very obvious if you misspell one of these attributes in one of the child classes. For example, if you accidentally typed `@asssessment` this variable would just be assigned as nil and then passed to the next bit of code. If you used the getter and miss-typed it as `asssessment` you'd immediate get a `method not defined` error which is a lot easier to debug.